frontend/src/app/component/phenotypic-feature/age-control/date/date.component.html

```diff
@@ -0,0 +1,31 @@
+<table class="full-width" cellspacing="0">
+    <tr>
+      <td>
+        <mat-form-field class="full-width" appearance="fill">
+          <mat-label>Years</mat-label>
+          <input matInput [formControl]="yearFormControl" type="number" placeholder="Ex. 1990" min="1" [errorStateMatcher]="matcher">
```

Comment:
This is not a field for a date (e.g. date of birth), but for fields of the ISO 8601 duration element. In our case, the minimum legal year is `0`, definitely not 1990. Max value is ~120 for the people who live in Okinawa, Japan..

Reply:
I think we should leave the max value unbounded, and we can emit a warning if somebody enters some crazy value, like `500`. But it should not be an error, I think.